### PR TITLE
Implement team member card

### PR DESCRIPTION
**Pull request by KGhising**
-Implement team member card
Screenshot (318)

### CODE REVIEW

File: src/Pages/Home.jsx [hunk 1]
```diff
@@ -64,173 +64,52 @@ class Home extends React.Component {
...
-              </div>
+          <div className='our-team-section'>
+            <h1>Our Team</h1>
+            <div className='team-member row'>
+              <TeamMemberCard
+                imgLink='/public/images/anish.jpg'
+                name='Anish Silwal Khatri'
+                description='Anish Sliwal Khatri is a Programming Genius and currently working at Leapfrog Technology. He is pursuing 
+                  his bachelors degree in CSIT at Bhaktapur Multiple Campus. He has a huge
+                  interest on Football and Game Development. He is real blue supporter.'
```
Comment (silwalanish) [inline src/Pages/Home.jsx:75]: ???
```suggestion
                description='Anish Sliwal Khatri is a Software Engineer at Leapfrog Technology. He is pursuing 
                  his bachelors degree in BSc.CSIT at Bhaktapur Multiple Campus. He has a huge interest in Game Development. He is a massive
                  football fan (support Chelsea Football Club).'
```

File: src/Themes/TeamMemberCard.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: map-get($map: $theme-colors, $key: 'white');
+  border-radius: 16px;
+  font-size: 14px;
+
+  p {
+    display: none;
+    text-align: left;
```
Comment (silwalanish) [inline src/Themes/TeamMemberCard.scss:15]:
```suggestion
    text-align: justify;
    letter-spacing: something_a_bit_readable;
    text-transform: uppercase; // I think uppercased would look better.
```

File: src/Themes/TeamMemberCard.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  max-width: 260px;
+  height: 300px;
+  margin-top: 34px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: map-get($map: $theme-colors, $key: 'white');
+  border-radius: 16px;
+  font-size: 14px;
```
Comment (silwalanish) [inline src/Themes/TeamMemberCard.scss:11]: Add transition animation.

Comment (KGhising) [reply src/Themes/TeamMemberCard.scss:11]: Okay

**Comment by silwalanish** [comment]
@KGhising Fix conflicts as well.

File: src/Pages/Home.jsx [hunk 4]
```diff
@@ -76,6 +76,10 @@ class Home extends React.Component {
     );
     return (
       <React.Fragment>
+        <div className='home-page col-12'>
+          <div className='d-flex align-items-center' height='90vh'>
+            <div>
+              <h1>Construct your 3D models</h1>
```
Comment (silwalanish) [inline src/Pages/Home.jsx:82]: ???
```suggestion
```

File: src/Pages/Home.jsx [hunk 5]
```diff
@@ -122,173 +123,52 @@ class Home extends React.Component {
...
-              </div>
+          <div className='our-team-section'>
+            <h1>Our Team</h1>
+            <div className='team-member'>
+              <TeamMemberCard
+                imgLink='/public/images/anish.jpg'
+                name='Anish Silwal Khatri'
+                description='Anish Sliwal Khatri is a Software Engineer at Leapfrog Technology. He is pursuing 
+                  his bachelors degree in Bsc.CSIT at Bhaktapur Multiple Campus. He has a huge
+                  interest in Game Development. He is a massive football fan (supports Chelsea Football Club).'
```
Comment (silwalanish) [inline src/Pages/Home.jsx:134]: The text is to big. Makes the icon overflow.
```suggestion
                description='Anish Sliwal Khatri is a Software Engineer at Leapfrog Technology. He is pursuing 
                  his bachelors degree in Bsc.CSIT. He has a huge
                  interest in Game Development. He is a massive football fan (supports Chelsea Football Club).'
```

**Review by silwalanish — APPROVED** [review]

**Pull request merged**